<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpLoadDialog</class>
 <widget class="QDialog" name="UpLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_view">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>录制视频</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>751</width>
     <height>531</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>上传视频限制大小50MB，视频格式flv或mp4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_path">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_view_check">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>略缩图</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>上传进度</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>490</y>
       <width>471</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_beginUpload">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>490</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始上传</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_funny">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>179</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>搞笑</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_outdoor">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>294</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>户外</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_ennegy">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>235</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111111</width>
       <height>111111</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>正能量</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_food">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>207</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>美食</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_educate">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>322</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>教育</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_video">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>266</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>影视</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_dance">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>151</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>舞蹈</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_music">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>350</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>57</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>歌曲</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_gif">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>531</width>
       <height>291</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QScrollArea" name="listhobby">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>140</y>
       <width>151</width>
       <height>301</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>149</width>
        <height>299</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="le_hobbyshow" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="pb_addItem">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加标签</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
